src/dattobd.h

```diff
@@ -57,13 +57,18 @@ struct reconfigure_params{
 #define COW_INDEX_ONLY 1
 #define COW_VMALLOC_UPPER 2
 
+#define COW_VERSION_0 0
```

Comment:
erm cow version 0 or cow version 1? above you have this: 
if(dev->sd_cow->version > COW_VERSION_0){
so if this is not 1, the seq_printf will never happen. I might be reading it wrong.

Reply:
sorry I take that back, I'm an idiot. never mind

Reply:
cm->version = COW_VERSION_CHANGED_BLOCKS;  <-- I missed that.